### PR TITLE
fix #278830: generate timesig after section break

### PR DESCRIPTION
**Pull request by tobik**
This PR adds a generated time signature to the system header after a section break and after a HBox when *Create System Header* is enabled.

### CODE REVIEW

File: libmscore/layout.cpp [hunk 1]
```diff
@@ -3444,15 +3444,15 @@ System* Score::collectSystem(LayoutContext& lc)
                   ww  += rest * m->ticks().ticks() * stretch;
                   m->stretchMeasure(ww);
                   m->layoutStaffLines();
-                  }
+            }
```
Comment (Jojo-Schmitz) [inline libmscore/layout.cpp]: nope, we use banner style indentation

File: libmscore/layout.cpp [hunk 2]
```diff
@@ -3444,15 +3444,15 @@ System* Score::collectSystem(LayoutContext& lc)
...
             else if (mb->isHBox()) {
                   mb->setPos(pos + QPointF(toHBox(mb)->topGap(), 0.0));
                   mb->layout();
                   }
             else if (mb->isVBox())
                   mb->setPos(pos);
             pos.rx() += ww;
-            }
+      }
```
Comment (Jojo-Schmitz) [inline libmscore/layout.cpp]: ditto

Comment (Jojo-Schmitz) [inline libmscore/layout.cpp]: ditto

File: libmscore/layout.cpp (hunk 1 above)
Comment (tobik) [reply libmscore/layout.cpp]: VS auto format - grrrr

**Comment by anatoly-os** [comment]
@tobik are you around to address the summary from the https://musescore.org/en/node/278830#comment-900629 as well as fix the tests?

**Comment by tobik** [comment]
I am still not 100% happy with the proposed approach. A section break property seams a little exaggerated. And please excuse me, but I estimated this patch will never get merged. 

If you are still interested it this PR, I will do complete it.

**Comment by dmitrio95** [comment]
Archiving this for now, please feel free to reopen or open a new pull request for this issue.

**Pull request closed without merging**